Makefile

```diff
@@ -428,7 +454,7 @@ data/mb_chao_garden_japan.gba.lz: data/mb_chao_garden_japan.gba
 
 %.bin: %.aif ; $(AIF) $< $@
 
-$(ELF): $(OBJS) libagbsyscall
+$(ELF): $(OBJS) $(LDSCRIPT) libagbsyscall
```

Comment:
LDSCRIPT is not needed as a dep here.

Reply:
There was an reason I addded it, but seems like you're right! :D